modules/drivers/gnss/include/gnss/novatel_messages.h

```diff
@@ -176,6 +180,110 @@ enum class DatumId : uint32_t {
   WGS84 = 61,
 };
 
+struct BDS_Ephemeris {
+  uint32_t satellite_id; //ID/ranging code
+  uint32_t week;  //week number
+  double ura; //user range accurancy(metres). This is the evaluated URAI/URA lookup-table value
+  uint32_t health1; //Autonomous satellite health flag. 0 means broadcasting satellite is good, 1 means not
+  double tdg1; //Equipment group delay differential for the B1 signal(seconds)
+  double tdg2; //Equipment froup delay defferential for the B2 signal(seconds)
+  uint32_t aodc; //Age of data, clock
+  uint32_t toc; //Reference time of clock parameters
+  double a0; //Constant term of clock correction polynomial(seconds)
+  double a1; //Linear term of clock correction polynomial (seconds/seconds)
+  double a2; //Quadratic term of clock correction polynomial (second/seconds^2)
+  uint32_t aode; //Age of data, ephemeris
+  uint32_t toe; //Reference time of ephemeris parameters
+  double rootA; //Square root of semi-major axis (sqrt(metres))
+  double ecc; //Eccentricity (sqrt(metres))
+  double omega; //Argument of perigee
+  double delta_N; //Mean motion difference from computed value (radians/second)
+  double M0; //Mean anomaly at reference time (radians)
+  double omega0; //Longitude of ascending node of orbital of plane computed according to reference time(radians)
+  double rra; //Rate of right ascension(radians/second)
+  double inc_angle; //inclination angle at reference time(radians)
+  double idot; //Rate of inclination angle(radians/second)
+  double cuc; //Amplitude of cosine harmonic correction term to the argument of latitude(radians)
+  double cus; //Amplitude of sine harmonic correction term to the argument of latitude(radians)
+  double crc; //Amplitude of cosine harmonic correction term to the orbit radius(metres)
+  double crs; //Amplitude of sine harmonic correction term to the orbit radius(metres)
+  double cic; //Amplitude of cosine harmonic correction term to the angle of inclination(radians)
+  double cis; //Amplitude of sine harmonic correction term to the angle of inclination(radians)
+};
+static_assert(sizeof(BDS_Ephemeris) == 196, "Incorrect size of BDS_Ephemeris.");
+
+struct GLO_Ephemeris {
```

Comment:
Can we use protobuf for such pure date struct?

Reply:
No, it is the format defined by Novatel receiver.